codestodians/food-basket-front-end
# #154141622 Fix text component error

#### What does this PR do?
Fixes Text component error

#### Description of Task to be completed?
Fix the error that comes with the styling of the Text component 

#### How should this be manually tested?
- Check out into the `bg-text-component-error-154141622` branch.
- Run the app as well as tests to ensure everything is still in order.
#### Any background context you want to provide?
When trying to style the text component with `styled-components`, it throws an undefined error

#### What are the relevant pivotal tracker stories?
[#154141622](https://www.pivotaltracker.com/story/show/154141622)
#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
